logistics/scripts/modules/utility/xml-validation.psm1

```diff
@@ -75,6 +75,7 @@ function Invoke-XmlValidation {
     $xmlReaderSettings.ValidationFlags =
     [System.Xml.Schema.XmlSchemaValidationFlags]::ProcessInlineSchema -bor
     [System.Xml.Schema.XmlSchemaValidationFlags]::ReportValidationWarnings
+    $xmlReaderSettings.Schemas.XmlResolver = New-Object System.Xml.XmlUrlResolver
```

Comment:
There's more info in dotnet's repo [here](https://github.com/dotnet/runtime/issues/30054).